examples/pipelines/kritis-resources.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: pipeline.knative.dev/v1beta1
+kind: Resource
+metadata:
+  name: kritis-resources-sample
+  namespace: default
+spec:
+  resources:
+  - name: kritis-app-github-resource
+    type: git
+    params:
+    - name: url
+      value: https://github.com/grafeas/kritis
+    - name: revision
+      value: master
+  - name: kritis-test-github-resource
+    type: git
+    params:
+    - name: revision
+      value: master
+    - name: url
+      value: https://github.com/grafeas/kritis-test  
```

Comment:
This repository does not seem to exist. Was it subsequently deleted? Or not publicly readable?

Reply:
funny you should mention this, i just noticed it myself! in #217 it's being updated to use a repo that exists - I think this was supposed to be an example of a case where someone has decided to put their tests in a separate repo (not something I'd recommend!)